RecastDemo/Include/Sample.h

```diff
@@ -40,15 +40,31 @@ enum SampleToolType
 
 /// These are just sample areas to use consistent values across the samples.
 /// The use should specify these base on his needs.
-enum SamplePolyAreas
-{
-	SAMPLE_POLYAREA_GROUND,
-	SAMPLE_POLYAREA_WATER,
-	SAMPLE_POLYAREA_ROAD,
-	SAMPLE_POLYAREA_DOOR,
-	SAMPLE_POLYAREA_GRASS,
-	SAMPLE_POLYAREA_JUMP,
-};
+// enum SamplePolyAreas
+// {
+// 	SAMPLE_POLYAREA_GROUND,
+// 	SAMPLE_POLYAREA_WATER,
+// 	SAMPLE_POLYAREA_ROAD,
+// 	SAMPLE_POLYAREA_GRASS,
+// 	SAMPLE_POLYAREA_DOOR,
+// 	SAMPLE_POLYAREA_JUMP,
+// };
+/// Mask of the ceil part of the area id (3 lower bits)
+/// the 0 value (RC_NULL_AREA) is left unused
+static const unsigned char SAMPLE_POLYAREA_CEIL_MASK = 0x07;
```

Comment:
What is the mening of 'ceil'?
I think it would be clearer to refer to this as the type. So the area ID consists of type and flags.